fastlane/Fastfile

```diff
@@ -59,6 +59,95 @@ lane :github_release do |options|
   )
 end
 
+desc "Update dependencies to latest GitHub releases"
+lane :update_native_dependencies_to_latest do |options|
+  latest_ios_release = get_latest_github_release_name(repo_name: "RevenueCat/purchases-ios")
+  latest_android_release = get_latest_github_release_name(repo_name: "RevenueCat/purchases-android")
```

Comment:
I think we should filter for "latest within the same major", otherwise we might be auto-bumping into a breaking version

Reply:
true. I thought in that case we just wouldn't merge it, but it's true that the PR will basically have broken code. I am going to add that check

Reply:
Done in [`b7a5bac` (#178)](https://github.com/RevenueCat/purchases-hybrid-common/pull/178/commits/b7a5baca32136d03e9e89d0fee40c754c6138bf6)